packages/boxed-expression-component/showcase/index.tsx

```diff
@@ -94,10 +94,10 @@ const INITIAL_EXPRESSION: ExpressionDefinition = {
 
 //Defining global function that will be available in the Window namespace and used by the BoxedExpressionEditor component
 const beeGwtService: BeeGwtService = {
-  getDefaultExpressionDefinition(logicType: string): ExpressionDefinition {
+  getDefaultExpressionDefinition(logicType: string, dataType: string): ExpressionDefinition {
     return getDefaultExpressionDefinitionByLogicType(
       logicType as ExpressionDefinitionLogicType,
-      { dataType: DmnBuiltInDataType.Undefined } as ExpressionDefinition,
+      dataType as ExpressionDefinition,
```

Comment:
Just this casting here that I think is not right?

Reply:
I mean, that second argument should be either a string or a DmnBuiltInDataType, right?

Reply:
Done! :)